app/controllers/foreman_google/api/v2/apipie_extensions.rb

```diff
@@ -8,6 +8,8 @@ module ApipieExtensions
           param :compute_resource, Hash do
             param :key_path, String, desc: N_('Certificate path, for GCE only')
             param :zone, String, desc: N_('Zone, for GCE only')
+            param :project, String, desc: N_('Deprecated, for GCE only, project is automatically loaded from the JSON file')
+            param :email, String, desc: N_('Deprecated, for GCE only, email is automatically loaded from the JSON file')
```

Comment:
To me this reads this is only deprecated for gce but valid for other compute resources. How about we put the gce part at the beginning or at the end?